ArraySorting.cls

```diff
@@ -0,0 +1,96 @@
+public class ArraySorting{
+	public static void ascendingOrder(List<Integer> values){

```

Comment:
Why there are 2 method for `ascending` and `descending` this can be merge in once method, you add extra param to method which will decide whether method needs to be in `ascending` or `descending`

- [ ] Merge two methods into one